lib/play.js

```diff
@@ -0,0 +1,22 @@
+const { chooseOX, chooseUserGrid, chooseCpuGrid } = require("./choices");
+const { pickOrder, playOrder } = require("./playOrder");
+
+const play = async (store) => {
+  await chooseOX(store);
```

Comment:
move setTeam into a store.js file

`lib/store.js`:
```
const setTeam = (store, choice) => {
  store.userTeam = choice;
  if (choice === "O") {
    store.cpuTeam = "X";
  } else {
    store.cpuTeam = "O";
  }
};
```

and then use 
```
const team = await getTeamChoice()
setTeam(store, team);
```
here

and delete `storeTeamChoice` because it's a 2-liner anyway.